src/Compilers/Core/Portable/Symbols/Attributes/UnmanagedCallersOnlyAttributeData.cs

```diff
@@ -37,7 +37,7 @@ internal static bool IsCallConvsTypedConstant(string key, in TypedConstant value
         {
             return key == "CallConvs"
                    && value.Kind == TypedConstantKind.Array
-                   && value.Values.All(v => v.Kind == TypedConstantKind.Type);
+                   && (value.Values.IsDefaultOrEmpty || value.Values.All(v => v.Kind == TypedConstantKind.Type));
```

Comment:
>v.Kind == TypedConstantKind.Type [](start = 78, length = 32)

Do we want to treat null values in the array as illegal? Is this check going to do that?

Reply:
Null values in the array are illegal, [https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs#L1006](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs#L1006) checks it. It's tested here: [https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs#L8051](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs#L8051)

---
In reply to: [491007732](https://github.com/dotnet/roslyn/pull/47651#discussion_r491007732) [](ancestors = 491007732)